### DISCUSSION TITLE
neovim completion lua mapping

### DISCUSSION THREAD

**Discussion by AdamBD** [community/community#50415] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I am trying to create a lua mapping to remap the copilot completion to something other than tab.

```
keymap("i", "C-J", ":lua copilot_accept()", { noremap = true, silent = true })

function copilot_accept()
  return vim.fn["copilot#Accept"]("<CR>")
end

vim.g.copilot_no_tab_map = true
```

I am trying to use this but it doesnt work

**Comment by rphuber** [comment, marked as answer]
These settings have worked for me:
```
vim.g.copilot_no_tab_map = true
vim.g.copilot_assume_mapped = true

vim.api.nvim_set_keymap("i", "<C-f>", 'copilot#Accept("<CR>")', { silent = true, expr = true })
```

**Reply by cryptowww** [reply]
work well from me

**Comment by rphuber** [comment]
also if you are trying to use control-j you would want `<c-j>` or if you want control shift J you would want `<c-J>`

**Comment by AdamBD** [comment]
@rphuber thanks! super helpful.
